protocol/governance/tomls/proxy-base.toml

```diff
@@ -6,6 +6,7 @@ defaultValue = "governance"
 [contract.InitialModuleBundle]
 artifact = "contracts/modules/core/InitialModuleBundle.sol:InitialModuleBundle"
 create2 = true
+salt = "<%= settings.salt %>"
```

Comment:
should be same as one in `synthetix`

```toml
[contract.InitialModuleBundle]
artifact = "contracts/modules/InitialModuleBundle.sol:InitialModuleBundle"
salt = "<%= settings.bundleSalt !== 'main' ? settings.bundleSalt : settings.salt %>"
create2 = true
```

Reply:
fixed in [190d772](https://github.com/Synthetixio/synthetix-v3/pull/2288/commits/190d772ea55e66fbec25a16ebc3003ebcca253ac)